SAMv1.tex

```diff
@@ -247,16 +247,19 @@ \subsection{The header section}
   key is the {\sf RNAME} field, with order defined by the order of {\tt @SQ} lines in the header.  The
   minor sort key is the {\sf POS} field.  For alignments with equal {\sf RNAME} and {\sf POS}, order is
   arbitrary.  All alignments with `{\tt *}' in {\sf RNAME} field follow alignments with some other
-  value but otherwise are in arbitrary order.\\\cline{2-3}
+  value but otherwise are in arbitrary order.
+  For queryname sort, no explicit requirement is made regarding the ordering other than that it be applied consistently throughout the entire file.%
+  \footnote{It is known that widely used software libraries have differing definitions of the queryname sort order, meaning care should be taken when operating on multiple files of varying provenance.
+  Tools may wish to use the sub-sort field to explicitly distinguish between natural and lexicographical ordering.  See Section~\ref{sec:sub-sort}.}\\\cline{2-3}
   & {\tt GO} & Grouping of alignments, indicating that similar alignment records
     are grouped together but the file is not necessarily sorted overall.
     \emph{Valid values}: {\tt none} (default), {\tt query} (alignments are
     grouped by {\sf QNAME}), and {\tt reference} (alignments are grouped by
     {\sf RNAME}/{\sf POS}).\\\cline{1-3}
-  & {\tt SS} & Sub-sorting order of alignments. Valid values are of the form \emph{sort-order}{\tt :}\emph{sub-sort}, where \emph{sort-order} is the same value stored in the {\tt SO} tag and \emph{sub-sort} is an implementation-dependent colon-separated string further describing the sort order.%
+  & {\tt SS} & Sub-sorting order of alignments. Valid values are of the form \emph{sort-order}{\tt :}\emph{sub-sort}, where \emph{sort-order} is the same value stored in the {\tt SO} tag and \emph{sub-sort} is an implementation-dependent colon-separated string further describing the sort order, but with some controlled terms defined in Section~\ref{sec:sub-sort}.
```

Comment:
here we use "some controlled terms" but in the sub-sort section it is "implementation-defined keywords" and "predefined terms".  How about using the latter?

Reply:
The above text also uses implementation-dependent, so I'll change the controlled terms to predefined terms and the implementation-defined bit in the later section to implementation-dependent for consistency.  Thanks.